src/Graphics/Glyphs/GlyphGeom.cc

```diff
@@ -119,7 +119,7 @@ void GlyphGeom::addCylinder(const Point& p1, const Point& p2, double radius, int
                             const ColorRGB& color1, const ColorRGB& color2,
                             bool renderBase1, bool renderBase2)
 {
-  addCylinder(p1, p2, radius, radius, color1, color2, renderBase1, renderBase2);
+  addCylinder(p1, p2, radius, radius, resolution, color1, color2, renderBase1, renderBase2);
```

Comment:
Here you see the problem with multiple many-parameter overloads of the same function. These would be good to convert to builder pattern like with (some of the) widgets and glyphs you started already.

Reply:
I converted this one. There's clipping planes which calls addSphere and addCylinder but I think that one should call those functions instead of the builders directly. addSphere doesn't use the builder constructor because it is neither a vector or a tensor. I just left the function generateSphere to do that.